docs/tutorials/index.rst

```diff
@@ -61,6 +61,13 @@ use the `Cinnabar Python package <https://github.com/OpenFreeEnergy/cinnabar>`_
 to analyze (e.g. generating MLE estimates of absolute free energies)
 and plot networks of relative free energy results.
 
+Generating Partial Charges
+--------------------------
+
+The :any:`Generating Partial Charges CLI tutorial <charge_molecules_cli_tutorial>` demonstrates
+ how the command line interface can be used to assign and store partial charges for small molecules which
+can be used throughout the OpenFE eco-system.
+
```

Comment:
Should this be moved to the top of the list as we now recommend it as the first step in most workflows?

Reply:
Sounds good to me.